### DISCUSSION TITLE
HTTP request failed. Client is already closed.

### DISCUSSION THREAD

**Discussion by alientae** [emailjs-com/help#26] [Q&A]
I think the problem is at my template because I already checked my service_id, template_id, publicKey and privateKey, everything is correct.

I use my EmailJs email and name for my "From Name" and "From Email" fields. I also checked the box for "Default Email Address". For "To Email", i pass a variable {{email}} to get the email from my flutter code. Meanwhile for "Reply To" field, I just left it empty. I don't have any idea why I received this kind of error. Please help me.

Below is the sample of my template:
<img width="426" alt="image" src="https://github.com/emailjs-com/help/assets/136829178/0e2a7f1b-7099-46d3-b836-a538fd9b2766">

**Comment by xr0master** [comment]
This error is related to the Internet protocols. Where are you getting it?
